src/main.cpp

```diff
@@ -10,34 +10,28 @@
 
 #include <iostream>
 #include <string>
+#include <map>
 #include <boost/asio.hpp>
 #include <boost/bind.hpp>
 #include "server.hpp"
 #include "nginx-configparser/config_parser.h"
+#include "server_options.hpp"
 
-// Expected format of config file:
-// server {
-//   listen 8080;
-//   path /echo EchoHandler;
-//   path /static StaticFileHandler {
-//      root static;
-//   }
-// }
-
-struct server_options {
-  std::string port;
-  std::string static_file_root;
-  std::string echo_handler;
-  std::string static_handler;
-};
+namespace http{
```

Comment:
I was experimenting with this, but turns out main cannot be part of a namespace: 
http://stackoverflow.com/questions/3956678/main-in-namespace